tests/src/test/resources/kg/schemas/bbp/brain-region-ontology.json

```diff
@@ -0,0 +1,86 @@
+{
+  "@context": "https://neuroshapes.org",
+  "@id": "http://bbp.epfl.ch/neurosciencegraph/ontologies/core/brainregion_slim",
+  "@type": "Ontology",
+  "defines": [
+    {
+      "@id": "http://api.brain-map.org/api/v2/data/Structure/170",
+      "@type": "Class",
+      "label": "Dorsal part of the lateral geniculate complex",
+      "subClassOf": [
+        "https://neuroshapes.org/BrainRegion"
+      ]
+    },
+    {
+      "@id": "http://api.brain-map.org/api/v2/data/Structure/943",
+      "@type": "Class",
+      "label": "Primary motor area, layer 2/3",
+      "subClassOf": [
+        "https://neuroshapes.org/BrainRegion"
+      ]
+    }
+  ],
+  "distribution": [
```

Comment:
I don't understand why we need the distribution here? It's not important but it should work without it?

Reply:
You are right, I could have get rid of it here (it was with the original ontology which is way larger)